build/sync/plan.yml

```diff
@@ -42,3 +42,14 @@ actions:
         conditions:
           - type: exists
           - type: file_unaltered
+  - paths:
```

Comment:
Same question here. Is this only for testing?

Reply:
Yes, this should not be here.